AABB_tree/demo/AABB_tree/Scene.cpp

```diff
@@ -1319,12 +1321,8 @@ void Scene::deactivate_cutting_plane()
 }
 void Scene::initGL()
 {
-    gl = new QOpenGLFunctions_2_1();
-   if(!gl->initializeOpenGLFunctions())
-    {
-        qFatal("ERROR : OpenGL Functions not initialized. Check your OpenGL Verison (should be >=3.3)");
-        exit(1);
-    }
```

Comment:
Why have you removed the check for the return value of `initializeOpenGLFunctions`?

Reply:
because in QOpenGLFunctions() it is not a boolean anymore, only a void.